.github/scripts/docker/DockerFile

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/distroless/java-debian10
```

Comment:
I think `gcr.io/distroless/base-debian10` should work as a base, doesn't it?

Reply:
If I use `gcr.io/distroless/base-debian10`, the test is fail -
` /usr/local/bin/scala: error while loading shared libraries: libstdc++.so.6: cannot open shared object file: No such file or directory`

Reply:
The mostly-static binary is precisely made not to depend on `libstdc++`. It works fine [here](https://github.com/VirtuslabRnD/scala-cli/blob/c01a212af47e61a2c530f7bf0b4186c4a58bf991/modules/integration-core/src/test/scala/scala/cli/integration/RunTestDefinitions.scala#L809-L811).

Reply:
It's works. Locally, I was using different binary with scala-cli so it throws an error.